src/data/automation_i18n.ts

```diff
@@ -397,12 +397,10 @@ const tryDescribeTrigger = (
   }
 
   // Time Pattern Trigger
-  if (
-    trigger.platform === "time_pattern" &&
-    (trigger.seconds !== undefined ||
-      trigger.minutes !== undefined ||
-      trigger.hours !== undefined)
-  ) {
+  if (trigger.platform === "time_pattern") {
+    if (!trigger.seconds && !trigger.minutes && !trigger.hours) {
```

Comment:
I think this still should be undefined test and not a falsy test?

 `trigger.seconds` = 0 is a valid trigger that should be translated.

Reply:
If they are all 0, would that be valid? I guess, but weird...